cfpb/consumerfinance.gov
# Auto vanity URLs for payments-by-case children

This PR automatically redirects any URL starting with `/payments/` to `/enforcement/payments-harmed-consumers/payments-by-case/`, allowing for automatic generation of vanity URLs for the upcoming web-based Payments by Case pages that will replace existing PDFs now and moving forward.

---




## Additions

- New RedirectMatch in `redirects.conf`


## How to test this PR

1. localhost
2. Attempt to visit any URL string starting with /payments/, e.g. `localhost/payments/testing`
3. Confirm this redirects correctly, e.g. `/enforcement/payments-harmed-consumers/payments-by-case/testing`


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
